app/models/group.rb

```diff
@@ -5,11 +5,12 @@ class Group < ApplicationRecord
   has_many :groups_users
   has_many :users, through: :groups_users
   has_many :lists, as: :owner
+  has_many :resources, through: :lists
 
   validates :name, presence: true
 
-  def resources
-    lists.map(&:resources).flatten.uniq
+  def latest_resources(limit: 4)
+    resources.sort_by_created_at.distinct.limit(limit)
```

Comment:
I dig it. If I'm not mistaken, this change will move these operations to the database level, and not do them in memory, correct?